DittoPresenceViewer/build.gradle

```diff
@@ -3,10 +3,14 @@ plugins {
     alias libs.plugins.org.jetbrains.kotlin.android
 }
 
+ext.libraryGroupId = "live.ditto.dittopresenceviewer"
+ext.libraryVersion = "0.0.1"
```

Comment:
We should set an explicit `artifactId` to control the maven coordinate name in case this gradle module name ever changes.
https://github.com/getditto/DittoAndroidTools/blob/69ae98ea366873a6c22f2da0517e36c5b1c1a6a7/gradle/deploy.gradle#L42C22-L44

See the [`publish.gradle`](https://github.com/getditto/ditto/blob/main/android/ditto/publish.gradle#L11) used by the Android SDK.

Reply:
```suggestion
ext.libraryGroupId = "live.ditto"
ext.libraryArtifactId  = "presenceviewer"
ext.libraryVersion = "0.0.1"
```